hrms/hr/doctype/expense_claim/expense_claim.json

```diff
@@ -16,25 +16,39 @@
   "column_break_5",
   "expense_approver",
   "approval_status",
+  "currency_section",
+  "currency",
+  "column_break_imlz",
+  "exchange_rate",
   "expense_details",
   "expenses",
   "taxes_and_charges_sb",
   "taxes",
   "advance_payments_sb",
   "advances",
   "transactions_section",
+  "column_break_nexp",
+  "base_total_sanctioned_amount",
```

Comment:
Try to keep related fields beside each other? The current order is really hard to follow.

<img width="1210" height="899" alt="expense-claim-amt-fields-mismatch" src="https://github.com/user-attachments/assets/846dbb1f-6eed-4c20-8ae0-0245b5745778" />

It should be in the order of math being applied:

Claimed INR - USD
Sanctioned INR - USD
Taxes & Charges INR - USD
Total Advance Amount INR - USD
Grand Total INR - USD = (sanctioned + taxes - advances)
Reimbursed INR - USD